src/frontend/components/split-pane/split-pane.html

```diff
@@ -0,0 +1,23 @@
+<div
+   #wrapper
+   (window:resize)="windowResized()"
+   class="flex flex-auto">
+  <div
+     #primary
+     class="col overflow-hidden flex mh100pct">
+    <ng-content select="split-pane-primary-content"></ng-content>
+  </div>
+  <div
+     #secondary
+     class="col overflow-hidden flex mh100pct">
+    <ng-content select="split-pane-secondary-content"></ng-content>
+  </div>
+  <div
+     #resizer
+     (mousedown)="resizerMouseDown($event)"
+     style="width: 6px; top: 0; bottom: 0; margin-right: -3px; cursor: ew-resize; z-index: 500; position: absolute;">
```

Comment:
@xorgy I totally agree with you but I would still like to see the styles in CSS as seeing it in the code someone else might think we are using inline styles and start doing so I want to avoid that. It's not a big deal but just sticking to conventions is good.